Needed for HEAD and glibc-2_7-branch.

Reported on <http://savannah.gnu.org/bugs/?18218>.

This is not a proper fix, but a workaround.


Index: resolv/res_send.c
===================================================================
RCS file: /cvs/glibc/libc/resolv/res_send.c,v
retrieving revision 1.49
diff -u -p -r1.49 res_send.c
--- resolv/res_send.c	6 May 2006 17:35:50 -0000	1.49
+++ resolv/res_send.c	13 Jun 2006 13:57:51 -0000
@@ -102,6 +102,10 @@ static const char rcsid[] = "$BINDId: re
 #define MAXPACKET       65536
 #endif
 
+#ifndef MSG_NOSIGNAL
+# define MSG_NOSIGNAL	0
+#endif
+
 
 /* From ev_streams.c.  */
 
